test/Client.spec.js

```diff
@@ -777,4 +777,42 @@ describe("Client", () => {
       });
     });
   });
+
+  describe("Invitations", done => {
+    it("Get invitations", done => {
```

Comment:
@ydeshayes you can use async/await inside of the tests and avoid using the `done` callback.

https://github.com/platformsh/platformsh-client-js/blob/master/test/Client.spec.js#L733